### PR TITLE
Update Mosaic Class

### PR DESCRIPTION
**Pull request by kunicmarko20**
I am targeting this branch, because this is BC.

## Changelog

```markdown
### Changed
- allow using overwritten constant for the mosaic html classes inside of admin class
```

## Subject

There was a request for this feature on slack since const is used it seems reasonable.

### CODE REVIEW

File: src/Admin/AbstractAdmin.php [hunk 1]
```diff
@@ -2726,7 +2726,7 @@ public function getDashboardActions()
     final public function showMosaicButton($isShown)
     {
         if ($isShown) {
-            $this->listModes['mosaic'] = ['class' => self::MOSAIC_ICON_CLASS];
+            $this->listModes['mosaic'] = ['class' => static::MOSAIC_ICON_CLASS];
```
Comment (yceruto) [inline src/Admin/AbstractAdmin.php]: maybe here too to be consistent?
https://github.com/sonata-project/SonataAdminBundle/blob/56b0201c63c50e61e6bb002c78fdd1991924d347/src/Admin/AbstractAdmin.php#L452

Comment (kunicmarko20) [reply src/Admin/AbstractAdmin.php]: You can't use static when defining a default value for a property.

Comment (greg0ire) [reply src/Admin/AbstractAdmin.php]: what property? I don't understand what you mean

Comment (kunicmarko20) [reply src/Admin/AbstractAdmin.php]:
```
    protected $listModes = [
        'list' => [
            'class' => 'fa fa-list fa-fw',
        ],
        'mosaic' => [
            'class' => self::MOSAIC_ICON_CLASS,
        ],
    ];
```
https://3v4l.org/shiFi

**Comment by OskarStark** [comment]
Thank you @kunicmarko20 👍

**Pull request merged**